source/blog/2016-05-12-gaad.md

```diff
@@ -0,0 +1,91 @@
+
+[The complete list of the Ember A11y project efforts you can participate in is here.](https://docs.google.com/spreadsheets/d/1q4DkaNwH8mh7xZJa1TmrHNcFuFuWdQ80iG88c7N4QII/edit#gid=808967448) 
+
+### Community Addons
+
+This year we've partnered with a few community addons to work on improving their accessibility bonafides. [The complete list of the community addons we've partnered with and efforts you can participate in is here.](https://docs.google.com/spreadsheets/d/1q4DkaNwH8mh7xZJa1TmrHNcFuFuWdQ80iG88c7N4QII/edit#gid=0) If you happen to use any of these addons in your own applications this is an excellent way to start contributing.
+
+The simplest way to get started is to do an accessibility review of an addon for how well it works using assistive tech. If you discover things that don't work well with assistive technology file an issue with the `[a11y]` tag as a prefix. If you are unsure of where to start just [reach out](#toc_participate-in-the-community) and we can work with you!
+
+### Learning About Accessibility
+
+Accessibility is a tremendously broad space and includes everything from design to tooling to code. We've selected a few resources and presentations you can read to get started in the accessibility space.
+
+#### Experiences
+
+You can experience the web using assistive tech and see what it's like. Turn off your display and surf the web with a screen reader for an hour. Try using [NVDA](http://www.nvda-project.org/) on Windows, VoiceOver on [OS X](http://help.apple.com/voiceover/info/guide/10.11/) and [iOS](http://help.apple.com/iphone/9/#/iph3e2e4367) or [TalkBack on Android](https://support.google.com/accessibility/android/answer/6283677?hl=en). Or better yet, all of them!
+
+#### Resources
+
+- Resource 1
+- Resource 2
+- Resource 3
+- Resource 4
+
+#### Presentations
+
+- [Accessibility Mechanics for Web Applications](https://vimeo.com/163925627) by [Léonie Watson](https://twitter.com/LeonieWatson) given at [Ember London](http://emberlondon.com/).
+- [Building for Accessibility](https://www.youtube.com/watch?v=ok9v9-Tcq0o) by [Nathan Hammond](https://twitter.com/nathanhammond) and [Jennison Asuncion](https://twitter.com/jennison) (one of the co-founders of GAAD) given at [Wicked Good Ember](https://wickedgoodember.com/). [Followup blog post.](http://www.nathanhammond.com/building-for-accessibility)
+- Presentation 3
```

Comment:
If we can't find something better (but please do!) we can add my a11y-debt talk in.